go.mod

```diff
@@ -27,7 +28,7 @@ require (
 	github.com/tomasen/fcgi_client v0.0.0-20180423082037-2bb3d819fd19
 	github.com/valyala/fastjson v1.5.4
 	github.com/vmware/govmomi v0.22.2
-	golang.org/x/net v0.0.0-20191027093000-83d349e8ac1a // indirect; needed for freebsd/arm64
```

Comment:
@ilyam8 . I don't know why this package was removed when I was working on wireguard module. Should I include it?

Reply:
as you see it is needed for `freebsd/arm64`

try w/o it

```cmd
0 go.d.plugin (master)$ pwd
/Users/ilyam/Projects/golang/go.d.plugin

0 go.d.plugin (master)$ hack/go-build.sh freebsd/arm64
Building binaries for version: v0.24.0
Building freebsd/arm64
```